model/cluster_request.go

```diff
@@ -379,3 +379,75 @@ func NewProvisionClusterRequestFromReader(reader io.Reader) (*ProvisionClusterRe
 
 	return &provisionClusterRequest, nil
 }
+
+type CreateNodegroupsRequest struct {
+	Nodegroups                 map[string]NodeGroupMetadata `json:"nodegroups"`
+	NodeGroupWithPublicSubnet  []string                     `json:"nodegroup-with-public-subnet,omitempty"`
+	NodeGroupWithSecurityGroup []string                     `json:"nodegroup-with-sg,omitempty"`
+}
+
+// SetDefaults sets default values for nodegroups.
+func (request *CreateNodegroupsRequest) SetDefaults() {
+	for ng, meta := range request.Nodegroups {
+		if len(meta.InstanceType) == 0 {
+			meta.InstanceType = "m5.large"
```

Comment:
I keep seeing instance sizes around the code for defaults, shouldn't we put this in a default constant somewhere?

Reply:
@gabrieljackson, we have some items like this. Do we want to make them const?

Reply:
Yep, that's probably a good idea, but I am fine doing this as a follow-up.